helix-term/src/commands.rs

```diff
@@ -438,8 +438,11 @@ impl MappableCommand {
         reverse_selection_contents, "Reverse selections contents",
         expand_selection, "Expand selection to parent syntax node",
         shrink_selection, "Shrink selection to previously expanded syntax node",
-        select_next_sibling, "Select next sibling in syntax tree",
-        select_prev_sibling, "Select previous sibling in syntax tree",
+        select_next_sibling, "Select next sibling in the syntax tree",
+        select_prev_sibling, "Select previous sibling the in syntax tree",
+        select_all_siblings, "Select all siblings of the current node",
+        select_all_children, "Select all children of the current node",
+        select_all_children_in_selection, "Select all children of the current node that are contained in the current selection",
```

Comment:
this reminds me of #3007, I would like to potentially see this as somekind of mode (potentially under `ms..`, `mk..`, `mr..`, `mS..` as @the-mikedavis suggested). Then would could have a default binding for this in the future (doesn't need to happen in this pr )